.github/ISSUE_TEMPLATE/Bug_Report.yml

```diff
@@ -0,0 +1,126 @@
+labels:
+  - Issue-Bug
+  - Need-Review
+body:
+- type: checkboxes
+  attributes:
+    label: Prerequisites
+    options:
+    - label: Write a descriptive title.
+      required: true
+    - label: Make sure you are able to repro it on the latest version
+      required: true
+    - label: Search the existing issues.
+      required: true
+- type: textarea
+  attributes:
+    label: Summary
+    description: >-
+      Write a short description of the issue at a high-level.
+    placeholder: >-
+      I am experiencing a problem with X.
+      I think Y should be happening but Z is actually happening.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: >
+      List of steps, sample code, failing test or link to a project that reproduces the behavior.
+      Make sure you place a stack trace inside a code (```) block to avoid linking unrelated issues.
+    placeholder: |-
+      1. Create the following configuration document:
+
+         ```yaml
+         # repro.dsc.config.yaml
+         $schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/config/document.json
+         resources:
+         - name: repro
+           type: Test/Echo
+           properties:
+             output: expected value
+         ```
+
+      1. Call the `get` operation on the repro document:
+
+         ```sh
+         dsc config get --path ./repro.dsc.config.yaml
+         ```
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected behavior
+    render: console
+    placeholder: |
+      dsc config get --path ./repro.dsc.config.yaml
+      
+      results:
+      - name: repro
+        type: Test/Echo
+        result:
+          actualState:
+            output: expected value
+      messages: []
+      hadErrors: false
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Actual behavior
+    render: console
+    placeholder: |
+      # Enable debug tracing, which shouldn't contain any private data
+      dsc -l debug config get --path ./repro.dsc.config.yaml
+      
+      results:
+      - name: repro
+        type: Test/Echo
+        result:
+          actualState:
+            output: other value
+      messages: []
+      hadErrors: false
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Error details
+    description: Paste verbatim output from DSC if it returns an error
+    render: console
+- type: textarea
+  attributes:
+    label: Environment data
+    description: Paste verbatim output from `$PSVersionTable` below.
+    render: PowerShell
+    placeholder: PS> $PSVersionTable
+  validations:
+    required: true
+- type: dropdown
+  validations:
+    required: true
+  attributes:
+    label: Version
+    description: >-
+      Specify the version you're using. Run the `dsc --version` command to get
+      the current version.
+    options:
+      - Latest build from `main`
```

Comment:
If we still use `options` isn't that a fixed list that we have to keep updating?